R/pkg/R/column.R

```diff
@@ -151,6 +151,40 @@ setMethod("substr", signature(x = "Column"),
             column(jc)
           })
 
+#' startsWith
+#'
+#' Determines if entries of x start with string (entries of) prefix respectively,
+#' where strings are recycled to common lengths.
+#'
+#' @rdname startsWith
+#' @name startsWith
+#' @family colum_func
+#'
+#' @param vector of character string whose “starts” are considered
```

Comment:
I will add param name. The first line is for `x` and the second line is for `prefix`